Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -304,10 +318,6 @@ extension XCTestCase {
             state.currentStepLocation = nil
             state.currentStepDepth -= 1
         }
-        
-        XCTContext.runActivity(named: initialExpression) { (_) in
-            perform(expression: initialExpression)
-        }
```

Comment:
Hah - removing this in two PRs is going to cause you conflicts in the future ;)

Reply:
Yeah, I was working on several things at the same time and they all required this change but didn't want them to block each other 🤷‍♂️